controllers/csi-replicator/replicator_common.go

```diff
@@ -39,13 +39,16 @@ func shouldContinue(ctx context.Context, class *storageV1.StorageClass, driverNa
 		return false
 	}
 
-	// TODO: This should be removed when we start supporting RGs for SRDF Metro too
-	// Check for PowerMax SRDF Metro
-	if value, ok := class.Parameters["replication.storage.dell.com/RdfMode"]; ok {
-		if strings.ToUpper(value) == "METRO" {
-			log.V(common.InfoLevel).Info("Metro replication is not supported by Dell CSI Replication Controllers")
-			return false
-		}
+	// Check for PowerMax SRDF Metro and skip since it is only supported at the driver level
+	if value, ok := class.Parameters["replication.storage.dell.com/RdfMode"]; ok && strings.ToUpper(value) == "METRO" {
+		log.V(common.InfoLevel).Info("Metro replication is not supported by Dell CSI Replication Controllers")
+		return false
+	}
+
+	// Check for PowerStore Metro and skip since it is only supported at the driver level
+	if value, ok := class.Parameters["replication.storage.dell.com/mode"]; ok && strings.ToUpper(value) == "METRO" {
+		log.V(common.InfoLevel).Info("Metro replication is not supported by Dell CSI Replication Controllers")
+		return false
```

Comment:
```suggestion
	// Check for Metro and skip since it is only supported at the driver level
if value, ok := class.Parameters["replication.storage.dell.com/RdfMode"]; (ok && strings.ToUpper(value) == "METRO") || 
   (value, ok = class.Parameters["replication.storage.dell.com/mode"]; ok && strings.ToUpper(value) == "METRO") {
	log.V(common.InfoLevel).Info("Metro replication is not supported by Dell CSI Replication Controllers")
	return false
}

```